src/domain/usecases/ValidateGlassUploadsUseCase.ts

```diff
@@ -2,7 +2,7 @@ import { UseCase } from "../../CompositionRoot";
 import { Future, FutureData } from "../entities/Future";
 import { GlassUploadsRepository } from "../repositories/GlassUploadsRepository";
 
-const submission = {
+const upload = {
```

Comment:
(not part of your PR) When defining big objects like this, we typically specify the type at definition. So if there is some problem, it will be marked here and not when used. `const upload: GlassUploads = ...`

Reply:
Noted in glass team slack, will be addressed in upcoming tasks.